Userland/Libraries/LibMarkdown/ContainerBlock.cpp

```diff
@@ -68,9 +68,19 @@ RecursionDecision ContainerBlock::walk(Visitor& visitor) const
 }
 
 template<class CodeBlock>
-static bool try_parse_block(LineIterator& lines, Vector<NonnullOwnPtr<Block>>& blocks, Heading* current_section)
+static bool try_parse_block(LineIterator& lines, Vector<NonnullOwnPtr<Block>>& blocks, Heading* current_section, bool is_interrupting_paragraph)
```

Comment:
(Not blocking on this comment)

Generally we try to avoid bool arguments in favor of Yes/No enums for code clarity. But given the state of the rest of this library, it would probably be more confusing to be the only non-bool around :sweat_smile: 

A follow-up to modernize this code to actually follow our style would be nice I s'pose.